### ISSUE TITLE
Conversation command doesn't seem to work as expected

### ISSUE THREAD

**Issue by alexfu** [DTVD/rainbowstream#82]
If i'm not mistaken, I would have expected the `conversation` command to show all replies to a given tweet. 

Instead (from reading https://github.com/DTVD/rainbowstream/blob/master/rainbowstream/rainbow.py#L465-L489) it grabs the `in_reply_to_status_id` property of the tweet, and if a value exists for it, grab the reply.

According to the Twitter docs for the `in_reply_to_status_id` property ... 

> If the represented Tweet is a reply, this field will contain the integer representation of the original Tweet’s ID.

Thus, making the `conversation` command only show replies to a tweet **if you give it an ID of a tweet that was in reply to the original tweet in question**. 

For example, consider the following tweet/status...

```
Google Play @GooglePlay an hour ago
  ₪:15 ♥:30 id:4 via Sprinklr
  If Google searches determined the @goldenglobes, @ImitationGame would win Best Drama. http://goo.gl/8mXH51 http://t.co/v1dwKEjamH
```
If I wanted to view replies to this, I would execute `conversation 4`...

```
[@alexfu914]: conversation 4

  Google Play @GooglePlay an hour ago
  ₪:19 ♥:32 id:4 via Sprinklr
  If Google searches determined the @goldenglobes, @ImitationGame would win Best Drama. http://goo.gl/8mXH51 http://t.co/v1dwKEjamH
```
In the above scenario, I would expect to see replies, but in reality I only see the original tweet.

By viewing the status in the web interface (https://twitter.com/GooglePlay/status/554276555384700929) I can see that someone had actually replied to that tweet.

Now, by going back to `rainbowstream` I can view the timeline of the user who had replied to `@GooglePlay` and grab the `rainbowstream` ID of the status.

```
[@alexfu914]: view @isteadde

  Dan Istead @isteadde an hour ago
  ₪:0 ♥:0 id:34 via Twitter Web Client
  @GooglePlay @goldenglobes @ImitationGame saw imitation game last night. good acting but script is too predictable.

[@alexfu914]: conversation 34

  Google Play @GooglePlay an hour ago
  ₪:18 ♥:32 id:4 via Sprinklr
  If Google searches determined the @goldenglobes, @ImitationGame would win Best Drama. http://goo.gl/8mXH51 http://t.co/v1dwKEjamH

  Dan Istead @isteadde an hour ago
  ₪:0 ♥:0 id:34 via Twitter Web Client
  @GooglePlay @goldenglobes @ImitationGame saw imitation game last night. good acting but script is too predictable.
```
Now this is the result I would have expected to see originally.

**Comment by DTVD** [comment]
Hi @alexfu Thanks for your idea :+1: 
The `conversation` command is not for the behavior you described.

For example We have this tweet 
![origin](http://i.gyazo.com/947ce817f4c03ec0897ce98963af554a.png)

And after click to the link `View conversation` I will have full thread from the beginning 
![conversation](http://i.gyazo.com/6f365f7469767f6d49680b68842092fd.png)

Your desired command is not implemented at this time. I'm not sure if it can be implemented with Twitter's API 1.1 (Twitter web interface is different with Twitter's API). 

A PR will be a great help :)

**Comment by alexfu** [comment]
@DTVD Yeah, after looking through the Twitter API docs, I was pretty disappointed to see that there is no API to retrieve replies to a tweet. However I have seen third party clients implement this feature. I'll do some more research and attempt to come back with a PR.

**Comment by alexfu** [comment]
@DTVD I have a working draft of this feature. Do you think this should be a new command? Currently in my draft it's part of the `conversation` command where if the tweet is not a reply, it will search for all replies to that tweet, otherwise it will execute the original code.

**Comment by DTVD** [comment]
Thanks for your help. I am just on vacation in a country with poor internet connection :cry: 

I think the new command should be separated with a name like `allrep`. If you don't sure about the implement, just throw a PR and we will discuss directly on the code :smile:
